### PR TITLE
add methods that send alert to the user

### LINKED ISSUE #96: Add job post mail alert method
add method that send alert to the users email if there are new job posts that suits their requirements
@advapi

### PR DESCRIPTION
**Pull request by advapi**
Add methods that:
- compare between last alert check and job post creation date
- send email to the user if there are new relevant job posts
- change test file name according to the test.ini file
- all the methods were checked manually by python shell- tests will be uploaded soon
fixes: #96

### CODE REVIEW

File: user_notification/models.py [hunk 1]
```diff
@@ -24,22 +26,58 @@ class JobAlert(models.Model):
     def __str__(self):
         return str(self.user_account_id)
 
-    # Calculate the date of the next alert
     @classmethod
     def calc_check_date(cls, last_check_date, frequency_in_days):
```
Comment (bardielle) [inline user_notification/models.py]: Change the function name to be more inductive

File: user_notification/models.py [hunk 2]
```diff
@@ -24,22 +26,58 @@ class JobAlert(models.Model):
...
+        send_mail(
+            'job alert from joboard',
+            'Hi, there are new jobs that suits your requirements! you can check them out in the website',
+            settings.EMAIL_HOST_USER,
+            [user_mail],
+            fail_silently=False,
+        )
+
+    @classmethod
+    def check_new_jobs(cls):
```
Comment (bardielle) [inline user_notification/models.py]: change the function name - you're not checking you're also doing something here...

File: user_notification/models.py (hunk 1 above)
Comment (advapi) [reply user_notification/models.py]: fixed @bardielle  @guy9050

File: user_notification/models.py (hunk 2 above)
Comment (advapi) [reply user_notification/models.py]: fixed @bardielle  @guy9050

**Pull request merged**